RLTS/SendEmail/etc/module.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="RLTS_SendEmail" setup_version="1.0.0" />
```

Comment:
Overcomplicated code - setup version is not needed, never versions of Magento use patch files; even so, this module does not have any setup scripts, etc.